types/three/three-core.d.ts

```diff
@@ -1656,7 +1656,7 @@ export class Object3D extends EventDispatcher {
     /**
      * An object that can be used to store custom data about the Object3d. It should not hold references to functions as these will not be cloned.
      */
-    userData: {[key: string]: any};
+    userData: {[key: string]: any} | any;
```

Comment:
This is just the same as `any`. Why write it this way instead? Please add a test that shows what you're trying to accomplish

Reply:
Because this data should be of any type, such as:
Let line:Line3 = new Line3();
line.userData = 1;//ok
line.userData = "test";//ok
line.userData = {data:"test"};//ok